### PR TITLE
add extra tests

### PR DESCRIPTION
**Pull request by pjfanning**
based on comments in #26 

fyi @vy

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -18,8 +18,8 @@
 root = true
 
 [*]
-indent_size = 4
-ij_continuation_indent_size = 8
+indent_size = 2
+ij_continuation_indent_size = 4
```
[the remark this answers is not in the document]
Comment (vy) [reply .editorconfig:22]: @pjfanning, can we limit this to Scala files, please? (We follow a similar _"additive"_ convention in `log4j-kotlin` too.)

**Review by vy — CHANGES_REQUESTED** [review]

**Comment by vy** [comment]
@pjfanning, thanks so much! :bow:

**Comment by vy** [comment]
@pjfanning, you need to sign your commits. Could you do that, please? Otherwise, I cannot (am not allowed to) merge it.

**Comment by pjfanning** [comment]
I have opened an issue with GitHub - it could take a while. They are marking my signed commits as unverified and it's not clear why. I will close this. Feel free to reuse my code yourself. I place this code in the public domain.

**Comment by vy** [comment]
Thanks @pjfanning! Cherry-picked your stuff in 468ab8cb1f4380043339fa1e0f0f5e43dfd154b2.

**Pull request closed without merging**